workflowai/core/client/agent.py

```diff
@@ -486,12 +487,22 @@ async def list_models(self) -> list[ModelInfo]:
         Raises:
             ValueError: If the agent has not been registered (schema_id is None).
         """
+
         if not self.schema_id:
             self.schema_id = await self.register()
 
-        response = await self.api.get(
+        data = ListModelsRequest()
```

Comment:
there is no validation if you create the model like this you should just use the constructor ?

Reply:
Changed in https://github.com/WorkflowAI/workflowai-py/pull/66/commits/fdbe09a041b055fac3fe119e1d8bd1412a5feda6